### DISCUSSION TITLE
Aula 25 - Refatorando os Testes

### DISCUSSION THREAD

**Discussion by rikemorais** [balta-io/7196#33] [Perguntas e Respostas] [answered]
Bom dia Teacher! Desculpa enviar dúvida no Carnaval, mas não precisa responder hoje.

Eu percebi que quando você executa o `dotnet test` o terminal retorna para você com essa tela:
![image](https://user-images.githubusercontent.com/42439687/156008734-fb7b3b14-7cd8-4000-b983-285fae708546.png)

Mas quando eu executo aparece essa:
![image](https://user-images.githubusercontent.com/42439687/156008947-c73e0619-d1ea-4296-b53e-37de11bff9ea.png)

Não sei o que posso ter esquecido. Vou continuar pesquisando para saber o que pode ser. Se eu descobrir antes eu posto aqui.

Abraço e bom Carnaval.

**Comment by andrebaltieri** [comment, marked as answer]
Boa tarde, @RikeMorais, como vai?

Será que você criou um projeto do tipo `mstest`? Em adicional, suas classes estão decoradas com `[TestClass]`?

**Reply by rikemorais** [reply]
Foi isso mesmo Master! Obrigado. Refiz e deu certo!

**Reply by andrebaltieri** [reply]
💜
